package/gluon-xlat464-clat/Makefile

```diff
@@ -0,0 +1,15 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-xlat464-clat
```

Comment:
The RFC 6877 calls it 464XLAT , maybe we should adhere to this order in naming?

Reply:
I agree.